tests/cover/test_health_checks.py

```diff
@@ -177,6 +177,35 @@ def a(x):
         a()
 
 
+def test_a_very_slow_test_will_fail_a_health_check():
+    called = [False]
+
+    @given(st.integers())
+    def a(x):
+        called[0] = True
```

Comment:
Hmm. I think I wanted to distinguish this from the case where it failed in the initial health check, though really there's no way it should do that here.